### DISCUSSION TITLE
pyrosim minimum density and HRR

### DISCUSSION THREAD

**Discussion by Alin1090** [firemodels/fds#12212] [General]
Hello, I'm working with Pyrosim and I had meet this warning: minimun density. 0.119 kg/m3，clipped in Mesh 6.
And the HRR I calculated was supposed to peak at about 460s, but it peaked at 200s and quickly dropped to 0. How can i solve those problems?
Thank you in advance!

**Comment by drjfloyd** [comment]
Converted to discussion.

Without an input file there is no way to know for sure; however, fires require air to burn and your symptoms suggest that your inputs are not providing air to continue the fire.
